bip-post-quantum-migration.mediawiki

```diff
@@ -0,0 +1,174 @@
+<pre>
+  BIP: TBD
+  Title: Post Quantum Migration and Legacy Signature Sunset
+  Layer: Consensus (soft fork)
+  Author: Jameson Lopp <jameson@lopp.net>
+          Christian Papathanasiou <cp@gcx.io>
+          Ian Smith <ianquantum2027@gmail.com>
+          Joe Ross <joeross7878@gmail.com>
+          Steve Vaile <steve@qsecdef.com>
+          Pierre-Luc Dallaire-Demers <pierre-luc@pauli.group>
+  Comments-Summary: No comments yet.
+  Comments-URI: TBD
+  Status: Draft
+  Type: Standards Track
+  Created: 2025-07-14
+  License: BSD-3-Clause
+  Requires: P2QRH as described in BIP-360
+</pre>
+
+== Introduction ==
+
+=== Abstract ===
+
+This proposal follows the implementation of post-quantum (PQ) output type (P2QRH) and introduces a pre-announced sunset of legacy ECDSA/Schnorr signatures. It turns quantum security into a private incentive: fail to upgrade and you will certainly lose access to your funds, creating a certainty where none previously existed.
```

Comment:
Note that BIP 360 currently describes a script tree based output type that differs from P2TR by not having the key path option, i.e., restricting itself to script path spending, but the leaf scripts still rely on BIP 340 Schnorr signatures. My understanding is that the authors of BIP 360 intend to produce another BIP that specifies the addition of PQ-signature schemes.

Reply:
Gotcha. I think to match current spec it would be more accurate to say only allow sending to segwit v3 outputs?

Reply:
Given that BIP 360 is still in flux, it’s either way probably still subject to future change

Reply:
Updated to v2 outputs for now.